web/components/manalinks/create-links-button.tsx

```diff
@@ -87,22 +115,27 @@ function CreateManalinkForm(props: {
           <div className="flex flex-col flex-wrap gap-x-5 gap-y-2">
             <div className="form-control flex-auto">
               <label className="label">Amount</label>
-              <input
-                className="input input-bordered"
-                type="number"
-                value={newManalink.amount}
-                onChange={(e) =>
-                  setNewManalink((m) => {
-                    return { ...m, amount: parseInt(e.target.value) }
-                  })
-                }
-              ></input>
+              <div className="relative">
+                <span className="absolute mx-3 mt-3.5 text-sm text-gray-400">
```

Comment:
I _think_ I like the way this is all inline: 
![image](https://user-images.githubusercontent.com/856034/179033748-f96c624f-6d9c-4d99-bb88-0aaa576200e2.png)

But also it looks just a little bit odd to me being grayed out like that. Maybe because it's the only instance of gray-400 on the form; the placeholder text "From AustinChen" is a different shade too.

Also note that this is still inconsistent with the Amount field we show on our bet panel 
![image](https://user-images.githubusercontent.com/856034/179034066-58c4f2d2-a76a-4e3b-afcf-54a1ef8e06a9.png)

and we should maybe consolidate at some point?

Reply:
I was noticing yesterday that our inputs are not rounded, but I think they should be. Some consolidation would be good! I'm not sure whether I prefer the gray background or not.

Reply:
Will round other inputs, the poll I made shall chose which M$ style :)